tools/readme.md

```diff
@@ -8,25 +8,28 @@ This tool will re-deploy your resources using the assets generated by Azure Dev
 3. If required, deploys an ingress controller to provide a public endpoint to access the service over the internet.
 
 ## Prerequisites
-Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash [cloudshell](https://shell.azure.com/ ) for a seamless experience.
+Please install the below pre-requisites if using this on your local machine, or alternately, you can just use azure bash [cloudshell](https://shell.azure.com/) for a seamless experience.
```

Comment:
Nit: please consider replacing `azure bash` (not sure where it's coming from?) by `Azure Cloud Shell`.

Reply:
Sure, I will update it to be more clear.